pex/environment.py

```diff
@@ -22,6 +22,11 @@
 from pex.tracer import TRACER
 from pex.util import CacheHelper, DistributionHelper
 
+if "__PEX_UNVENDORED__" in __import__("os").environ:
```

Comment:
The PEXEnvironment class is only used at runtime and at runtime a PEX only has access to the Pex code needed to bootstrap the runtime (scrub the sys.path). The bootstrap code is contained in `.bootstrap/` in the PEX and that's controlled here:
https://github.com/pantsbuild/pex/blob/a7d98116daedcacda4716ebc9ca831303f24b71f/pex/pex_builder.py#L436-L444

So you'd need to add `'wheel'` to the `root_module_names` list.

That said, I don't think this is the way to go. See below...